server.js

```diff
@@ -19,8 +36,43 @@ app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// app.get('/', (req, res) => {
+//   res.send('Happy Thoughts')
+// })
+
+app.get('/', async (req, res) => {
+  const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec()
+  console.log("Thoughts array")
+  res.json(thoughts)
+})
+
+app.post('/thoughts', async (req, res) => {
+  const {message} = req.body
+  const thought = new Thought({message})
+ 
+  try {
+    const savedThought = await thought.save()
+    res.status(201).json(savedThought)
+    console.log("Hit Success?")
+  }catch (err){
+    res.status(400).json({message: 'Could not save thought', error: err.errors})
+    console.log("In chatch")
+  }
+})
+
+app.post('/:thoughtId/like', async (req, res) => {
+  try{ 
+    const like = await Thought.findOneAndUpdate(
+      { "_id": req.params.thoughtId },
+      { $inc: {"hearts": 1} }
+      //Lägg in new: true
+    )
+    console.log("Likes success")
```

Comment:
This output is really useful for debugging, not so useful when in a production environment